ocean_lib/ocean/test/test_ocean_assets.py

```diff
@@ -38,43 +42,166 @@ def test_register_asset(publisher_ocean_instance, publisher_wallet, consumer_wal
     assert ocn.assets.resolve(invalid_did) is None
 
 
-@pytest.mark.skip(reason="TODO: update function on OceanAssets class")
-def test_update(publisher_ocean_instance, publisher_wallet, consumer_wallet, config):
-    ocn = publisher_ocean_instance
-    block = ocn.web3.eth.block_number
-    alice = publisher_wallet
-    bob = consumer_wallet
+def test_update(publisher_ocean_instance, publisher_wallet, config):
+    """Helper function for asset update based on ddo_sa_sample.json."""
+    ocean = publisher_ocean_instance
+
+    _, erc20_token = deploy_erc721_erc20(
+        publisher_ocean_instance.web3, config, publisher_wallet, publisher_wallet
+    )
+
+    ocean = publisher_ocean_instance
 
-    ddo = create_asset(ocn, alice, config)
+    data_provider = DataServiceProvider
+    file1_dict = {"type": "url", "url": "https://url.com/file1.csv", "method": "GET"}
+    file1 = FilesTypeFactory(file1_dict)
+    encrypt_response = data_provider.encrypt(
```

Comment:
there is an encrypt utility on ocean_assets itself. Maybe we can use that for some concision.